### PR TITLE
[CMSP-319] Refrence sites-yml-validator util in WPMS search-replace docs

### PR DESCRIPTION
**Pull request by pwtyler**
## Summary
[WPMS Search Replace](https://docs.pantheon.io/guides/multisite/search-replace/) - Adds direction to the sites-yml-validator tool.

### Dependencies and Timing

- [x] Move pantheon-systems/sites-yml-validator to be a public repo

**Release**:
- [x] When ready

--------------------------------------------------

## Post Launch

**Do not remove** - To be completed by the docs team upon merge:

- [ ] Redirect `/old-path/` => `/new-path/` (if applicable)
- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)
- [ ] For Heroes - add a props post to the [discussion board](https://discuss.pantheon.io/c/pantheon-platform/documentation/17).
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Review by jazzsequence — APPROVED** [review]

**Comment by whitneymeredith** [comment]
[Preview](https://pr-8329-documentation.appa.pantheon.site/guides/multisite/search-replace/#subdomain-wpms)

**Review by whitneymeredith — APPROVED** [review]
LGTM @scottbuscemi I'll hold off on merging until your approval.

**Comment by scottbuscemi** [comment]
@whitneymeredith Good to go :)

**Pull request merged**
